src/content/en/fundamentals/engage-and-retain/push-notifications/notification-behaviour.md

```diff
@@ -0,0 +1,255 @@
+          console.log('User ❤️️\'s coffee.');
+          break;
+        case 'doughnut-action':
+          console.log('User ❤️️\'s doughnuts.');
+          break;
+        case 'gramophone-action':
+          console.log('User ❤️️\'s music.');
+          break;
+        case 'atom-action':
+          console.log('User ❤️️\'s science.');
+          break;
+        default:
+          console.log(`Unknown action clicked: '${event.action}'`);
+          break;
+      }
+    });
+
+![Logs for action button clicks and notification click.](./images/notification-screenshots/action-button-click-logs.png){: .center-image }
+
+### Tag
+
+The *tag* option is essentially a string ID that "groups" notifications together, providing
+an easy way to determine how multiple notifications are displayed to the user. This is easiest
+to explain with an example.
+
+Let's display a notification and give it a tag, of
+'message-group-1'. We'd display the notification with this code:
+
+        const title = 'Notification 1 of 3';
+        const options = {
+          body: 'With \'tag\' of \'message-group-1\'',
+          tag: 'message-group-1'
+        };
+        registration.showNotification(title, options);
+
+Which will show our first notification.
+
+![First notification with tag of message group 1.](./images/notification-screenshots/desktop/chrome-first-tag.png){: .center-image }
+
+Let's display a second notification with a new tag of 'message-group-2', like so:
+
+            const title = 'Notification 2 of 3';
+            const options = {
+              body: 'With \'tag\' of \'message-group-2\'',
+              tag: 'message-group-2'
+            };
+            registration.showNotification(title, options);
+
+ This will display a second notification to the user.
+
+![Two notifications where the second tag is message group 2.](./images/notification-screenshots/desktop/chrome-second-tag.png){: .center-image }
+
+Now let's show a third notification but re-use the first tag of 'message-group-1'. Doing this
+will close the first notification and replace it with our new notification.
+
+            const title = 'Notification 3 of 3';
+            const options = {
+              body: 'With \'tag\' of \'message-group-1\'',
+              tag: 'message-group-1'
+            };
+            registration.showNotification(title, options);
+
+Now we have two notifications even though `showNotification()` was called three times.
+
+![Two notifications where the first notification is replaced by a third notification.](./images/notification-screenshots/desktop/chrome-third-tag.png){: .center-image }
+
+The `tag` option is simply a way of group messages like this so that any old notifications that
+are currently displayed will be closed if they have the same tag as a new notification.
+
+A subtlety to using `tag` is that the browser will replace any old notification without any
+sound and vibration that would normally be played for a new notification.
```

Comment:
I think this sentence might make readers say, 'huh'? Try this:

A subtlety to using `tag` is that when replacing a notification, the browser does _not_ use sound and vibration.